client/src/pages/Login/DemoLogin/DemoLogin.tsx

```diff
@@ -0,0 +1,55 @@
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Formik, FormikHelpers } from 'formik';
+import useStyles from './useStyles';
+
+interface Props {
+  handleSubmit: (
+    {
+      email,
+      password,
+    }: {
+      email: string;
+      password: string;
+    },
+    {
+      setStatus,
+      setSubmitting,
+    }: FormikHelpers<{
+      email: string;
+      password: string;
+    }>,
+  ) => void;
+}
+
+export default function DemoLogin({ handleSubmit }: Props): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Formik
+      initialValues={{
+        email: 'testuser@gmail.com',
+        password: 'testpassword',
+      }}
+      onSubmit={handleSubmit}
+    >
+      {({ handleSubmit, isSubmitting }) => (
+        <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <Box textAlign="center" marginTop={5}>
+            <Button
+              type="submit"
+              size="large"
+              variant="contained"
+              color="primary"
+              className={classes.submit}
+              disableElevation
+            >
+              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Demo Login'}
```

Comment:
We should avoid inline styles in general, always feel free to if you see some used in the starter code. We can usually use the [`sx` prop](https://mui.com/system/the-sx-prop/#main-content) or ideally, we can use the props defined for the components. For example, we can try the [`color` prop](https://mui.com/api/circular-progress/#props) for `CircularProgress` here, `inherit` could work well.

Reply:
this circular progress component was taken from the starter code which also had inline styling but I understand we don't want to avoid them , thank you.